### PR TITLE
Switch operator to use protobuf client for non-CR resources

### PR DESCRIPTION
**Pull request by mfojtik**
This builds on https://github.com/openshift/library-go/pull/233 and switches the controller loops that deal with non-CR resources to client-go with protobuf enforced.

/cc @deads2k 
/cc @smarterclayton 

**NOTE:** Remove the artificial bump before merging.

### CODE REVIEW

**Comment by soltysh** [comment]
Surprisingly fast green, do you have data before and after this change?

**Comment by mfojtik** [comment]
Only network and DNS are present in clusteroperators so this must be some networking flake.

/retest

File: pkg/operator/starter.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ import (
 )
 
 func RunOperator(ctx *controllercmd.ControllerContext) error {
-	kubeClient, err := kubernetes.NewForConfig(ctx.KubeConfig)
+	protoKubeClient, err := kubernetes.NewForConfig(ctx.ProtoKubeConfig)
```
Comment (deads2k) [inline pkg/operator/starter.go]: don't rename this

Comment (mfojtik) [reply pkg/operator/starter.go]: fixed.

**Comment by mfojtik** [comment]
Fixed client rename, LGTM per Slack from @deads2k

**Comment by mfojtik** [comment]
Re-applied LGTM as the only change was a panic fix in library-go.

**Pull request merged**